packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx

```diff
@@ -88,10 +88,12 @@ const ActivityDetails = (props: Props) => {
 
   const isOwner = viewerLowestScope === 'TEAM'
 
+  const isGlobalBannerEnabled = window.__ACTION__.GLOBAL_BANNER_ENABLED
+
   return (
     <div className='flex h-full w-full flex-col bg-white'>
       <div className='flex grow'>
-        <div className='mt-4 w-full grow'>
+        <div className={clsx('mt-4 w-full grow', isGlobalBannerEnabled && 'pt-6')}>
```

Comment:
-1 as we build out our app, it'd be pretty easy to forget to include this logic! Instead of making sure this is included in every child component, could we just keep it in `<Action/>`? We can wrap the `<Switch/>` with <div className={isGlobalBannerEnabled ? 'pt-6' : ''}>

Reply:
Maybe, I can try this and see if layout breaks.

Reply:
The problem is we have some components that use `100vh` so we’d probably at least have to touch those. I can see about reducing the scope to `<Action />` and any components using `100vh`